src/pages/History.js

```diff
@@ -95,4 +96,9 @@ const Artikel = styled.div`
 const Button = styled.button`
   width: 60px;
   margin: 30px auto;
+  background-color: #ff6666;
+  border-radius: 10px;
+  &:hover {
+    cursor: pointer;
```

Comment:
cursor: pointer on hover nicht sinnvoll

Reply:
hi Frank, thanks for you comments. You are right, i have changed it now!